### PR TITLE
firedm: Add version 2022.2.5

### LINKED ISSUE #5853: [Request] Add FireDM
python open source (Internet Download Manager) with multi-connections, high speed engine, based on python, LibCurl, and youtube_dl
https://github.com/firedm/FireDM

### PR DESCRIPTION
**Pull request by KaranKad**
Added FireDM version 2022.2.5

Closes #5853

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
- Is the app 32bit or 64bit?
- downloads.dat, thumbnails.dat, user_themes.cfg can be handled together (use a ForEach-Object loop)

**Comment by KaranKad** [comment]
> * Is the app 32bit or 64bit?

Its a neutral one because they say this in their release page 
> * Portable version for windows 32bit and 64bit, no installation needed, just extract to any folder and double click FireDM-GUI.exe file.

File: bucket/firedm.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "version": "2022.2.5",
+    "description": "Multi-connections download manager",
+    "homepage": "https://github.com/firedm/FireDM",
+    "license": "LGPL-3.0-only",
+    "url": "https://github.com/firedm/FireDM/releases/download/2022.2.5/FireDM_2022.2.5.zip",
+    "hash": "03CB296C3E54C2CEE80B16E1E91708250A7FEE6D338F9BE88ACF9414429210A0",
+    "extract_dir": "FireDM",
+    "pre_install": [
+        "'downloads.dat', 'thumbnails.dat', 'user_themes.cfg' | ForEach-Object {",
+        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" | Out-Null }",
```
Comment (rashil2000) [inline bucket/firedm.json:11]:
```suggestion
        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" -ItemType File | Out-Null }",
```

By default New-Item creates a folder.

**Pull request merged**